modules/admission-webhook-create.adoc

```diff
@@ -0,0 +1,375 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-controllers.adoc
+
+[id="admission-webhook-create_{context}"]
+= Creating an admission webhook
+
+This procedure outlines high level steps to deploy a webhook server as a DaemonSet and to deploy the webhook server as an aggregated API server. Deploying the webhook server as an aggregated API server allows other {product-title} components to communicate with the webhook using internal credentials and facilitates testing using the `oc` command. This also enables role based access control (RBAC) into the webhook and prevents token information from other API servers being disclosed to the webhook. The API server calls `kube-apiserver` with its fully authenticated service account. The aggregated API server then calls to the webhook using the request header certificate. The webhook verifies incoming client certificates through the published request header aggregation CA.
+
+[NOTE]
+====
+The link:https://github.com/openshift/kubernetes-namespace-reservation[kubernetes-namespace-reservation] repository provides a full end-to-end example of a webhook server and aggregated API server deployment. Note that this repository has not been fully tested or validated by Red Hat.
```

Comment:
Again, check on this with Eric, but if we do keep this statement in here, we might want to explicitly say that it's also not supported. Like "has not been fully tested and is not supported by Red Hat."

Reply:
I have now updated the note to reflect this. I will keep this item open until a decision has been made about the repo reference.

Reply:
Having reviewed this within the OpenShift Docs team, we have made a decision to comment out the repo reference and instead I have included a prerequisite around having an existing, published admission webhook server container. The document provides a useful high level overview of the steps to deploy such an existing image. Please let me know if you have any questions.